File: 20170522 FORTRAN/Task_1/Task_1/fun_u.f90
subroutine fun_u(epsilon_d,theta,u)
    USE point_para
    
    IMPLICIT NONE
    
    REAL(kind=8),intent(in) :: epsilon_d,theta
    REAL(kind=8),intent(out) :: u
    REAL(kind=8),external :: funqtheta,fun_F_x
    ! Local variables
    REAL(kind=8) :: q_theta,F_epsilond,A,B1,B2,lambda
    
    !!!!!!!!!!!!!  
    A = p_para(1)
    B1 = p_para(14)
    B2 = p_para(15)
    lambda = p_para(9)
    
    q_theta = funqtheta(theta,A,B1,B2)
    F_epsilond = fun_F_x(epsilon_d)
    u = lambda*F_epsilond/(lambda*F_epsilond+theta*q_theta)
    
    
    return
end subroutine

File: 20170620 Python/Functions/Task_1/fun_u.m
% Task #20170221
% Related to May16/Oct17,2016
% edited in Feb21,2017
%%
function u = fun_u(epsilon_d,theta,lambda,typen,...
    A,B1,B2,epsilon_u)
% parameter

q_theta=fun_q_theta(theta,A,B1,B2);

F_epsilond = fun_F_x(epsilon_d,typen,epsilon_u);

u = lambda*F_epsilond/(lambda*F_epsilond+theta*q_theta);

return